File: onebot-sdk/src/main/java/cn/evole/onebot/sdk/response/contact/UnidirectionalFriendListResp.java
package cn.evole.onebot.sdk.response.contact;

import com.google.gson.annotations.SerializedName;
import lombok.Data;

/**
 * @author cnlimiter
 */
@Data
public class UnidirectionalFriendListResp {

    @SerializedName("user_id")
    private long userId;

    @SerializedName("nickname")
    private String nickname;

    @SerializedName("source")
    private String source;

}

File: onebot/src/main/kotlin/sdk/response/contact/UnidirectionalFriendListResp.kt
package cn.evolvefield.onebot.sdk.response.contact

import com.google.gson.annotations.SerializedName

class UnidirectionalFriendListResp {
    @SerializedName("user_id")
    var userId = 0L
    @SerializedName("nickname")
    var nickname = ""
    @SerializedName("source")
    var source = ""
}
